### ISSUE TITLE
Documentation - show saving a query to a variable

### ISSUE THREAD

**Issue by jenweber** [thefrontside/bigtest#816]
I think this could be part of a Test Examples section of the documentation, or something similar.

When working on my first BigTest written from scratch, I found that I was repeating myself a lot with things like `TextField('something', {focused: true})`. I was surprised to learn that you can safely save this to a variable used in a test:

```js
const someInput = TextField('something', {focused: true})

export default test('Interactors with BigTest')
  .step(Page.visit('/'))
  .step(someInput).exists()

// etc
```

This was very cool and very surprising to me, after coming from other test frameworks that have lots of side effects and leaks from issues like this.

**Comment by jenweber** [comment]
@cowboyd please add or edit to this if you have more details/ideas.

**Comment by jnicklas** [comment]
Interestingly even this is technically safe, though IMO not something we should encourage:

``` typescript
const someInputExists = TextField('something', {focused: true}).exists()

export default test('Interactors with BigTest')
  .step(Page.visit('/'))
  .step(someInputExists)
```

**Comment by cowboyd** [comment]
Agree. In summary, we should make sure to emphasize that the reference to the interactor is safe to re-use, but that you should not re-use actions and assertions.
